src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -30,11 +30,31 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.Display.ByGeometryColor</oldName>
```

Comment:
Since this node is being renamed for the second time, if we also update `oldName` here to `Display.Display.ByGeometryColor`, we can deal with migration from the current node name to the new name otherwise this migration entry will take care of migrating only the nodes that are even older than the current ones.